website/docs/r/resource_pool.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_resource_pool"
+sidebar_current: "docs-vsphere-resource-resource-pool"
+description: |-
+  Provides a vSphere resource pool resource. This can be used to create and manage resource pools.
+---
+
+# vsphere\_resource\_pool
+
+The `vsphere_resource_pool` resource can be used to create and manage
+resource pools in standalone hosts or on compute clusters.
+
+For more information on vSphere resource pools, see [this
+page][ref-vsphere-resource_pools].
+
+[ref-vsphere-resource_pools]: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.resmgmt.doc/GUID-60077B40-66FF-4625-934A-641703ED7601.html
+
+## Example Usage
+
+The following example sets up a resource pool in a compute cluster which uses
+the default settings for CPU and memory reservations, shares, and limits. The
+compute cluster needs to already exist in vSphere.  
+
+```hcl
+variable "datacenter" {
+  default = "dc1"
+}
+
+variable "cluster" {
+  default = "cluster1"
+}
+
+data "vsphere_datacenter" "dc" {
+  name = "${var.datacenter}"
+}
+
+data "vsphere_compute_cluster" "compute_cluster" {
+  name          = "${var.cluster}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+resource "vsphere_resource_pool" "resource_pool" {
+  name                = "terraform-compute-cluster-test"
+  root_resource_pool_id = "${data.vsphere_compute_cluster.compute_cluster.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the resource pool.
+* `root_resource_pool_id` - (Required) The [managed object ID][docs-about-morefs]
+  of the root resource pool of the compute resource the resource pool is in.
```

Comment:
_Grammar_ this could probaly be re-worded. Suggested something like:

> `parent_resource_pool_id` - (Required) The managed object ID of the parent resource pool. This can be the root resource pool for a cluster or standalone host, or a resource pool itself.

Note the new naming of the field - see the appropriate comment for this field in the schema.